packaging/nnstreamer.spec

```diff
@@ -97,25 +97,19 @@ make %{?_smp_mflags}
 ./unittest_sink --gst-plugin-path=.
 popd
 pushd tests
-# The 'testAll.sh' script requires 6~7min to run armv7l binary files in the current CI server.
+# The ssat requires 6~7min to run armv7l binary files in the current CI server.
 # The timeout value is 10min as a heuristic value from our experience.
-timeout=600
-interval=60
 ssat &
 pid=$!
-while ((timeout > 0)); do
-    sleep $interval
-    if [[ ! $(ps | grep "$pid") ]]; then
-        break;
-    fi
-    (( timeout -= interval ))
-done
-if [[ $(ps | grep "$pid") ]]; then
-    kill $pid
-    echo "GBS is stopped because of './testAll.sh' timeout(10min)"
+(sleep 600 
```

Comment:
Please append comment  meaning & goal of this statement for maintenance. 
As we all know,  too slow time of unit-test will be continually repeated. :(

Reply:
updated